src/device/libpixy_m4/src/sdmmc.cpp

```diff
@@ -198,40 +201,42 @@ static bool init_card(uint32_t &boot_cnt)
     {
         // Both header blocks are invalid. Must be a new card. Reformat...
         printf(LOG_PREFIX "Both headers not valid. Formatting SD Card.\n");
-        boot_cnt = 0;
+        session_cnt = 0;
         return sdmmc_format();
     }
 
     if (header != NULL)
     {
-        // Increment the boot count and write it back to the oldest header block
-        header->boot_cnt++;
+        // Increment the session count and write it back to the oldest header block
+        header->session_cnt++;
 
         if (header == &headerA)
         {
-            if (write_header(HEADER_BLOCK_ID_B, header->boot_cnt) == false)
+            if (write_header(HEADER_BLOCK_ID_B, header->session_cnt) == false)
             {
                 printf(LOG_PREFIX "Failed to update HeaderB\n");
                 return false;
             }
         }
         else
         {
-            if (write_header(HEADER_BLOCK_ID_A, header->boot_cnt) == false)
+            if (write_header(HEADER_BLOCK_ID_A, header->session_cnt) == false)
```

Comment:
Can this be packed down into else if (...) with L221 above?.

Reply:
Done